lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -59,14 +58,20 @@ export class SilentFlowClient extends BaseClient {
         const environment = request.authority || this.authority.getPreferredCache();
         const cacheRecord = this.cacheManager.readCacheRecord(request.account, this.config.authOptions.clientId, requestScopes, environment);
 
-        if (this.isRefreshRequired(request, cacheRecord.accessToken)) {
```

Comment:
What was the reason for removing the helper function?

Reply:
Removed it because there was no easy way to reconcile refresh due to token expiration or refresh due to refreshIn time. I can add it back if it makes sense but I figured it was easier to just list it in the function itself.